web/modules/custom/rdf_entity/src/Plugin/Field/FieldWidget/FileSizeCalculatorWidget.php

```diff
@@ -0,0 +1,169 @@
+<?php
+
+namespace Drupal\rdf_entity\Plugin\Field\FieldWidget;
+
+use Drupal\Core\Entity\EntityFieldManagerInterface;
+use Drupal\Core\Field\FieldDefinitionInterface;
+use Drupal\Core\Field\FieldItemListInterface;
+use Drupal\Core\Field\WidgetBase;
+use Drupal\Core\Form\FormStateInterface;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\file\Entity\File;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+
+/**
+ * Plugin implementation of the 'file_size_calculator_widget' widget.
```

Comment:
* Please explain the purpose of the widget. It's already evident from the annotation that this is a plugin that has the "file_size_calculator_widget" machine name.
* It doesn't seem like this belongs in the RDF Entity module. This is project code.
* Tests are missing.